test/test/unit/test_link.py

```diff
@@ -265,54 +276,60 @@ def test_remove_outlet(self, outlet1, outlet2):
+    @patch(fqname(Inlet), spec=Inlet, _pull=pull_mock(object()))
     def xtest_non_iterable_raised(self, inlet1):
         logging.getLogger('databay.Link').setLevel(logging.ERROR)
-        link = Link([inlet1], [], timedelta(seconds=1), name='test_non_iterable_raised')
+        link = Link([inlet1], [], timedelta(seconds=1),
+                    name='test_non_iterable_raised')
         with self.assertRaisesRegex(TypeError, 'Inlets must return iterable'):
             link.transfer()
 
-    @patch(fqname(Inlet), spec=Inlet, _pull=pull_mock(DummyIterable())) # this rv will raise DummyException
+    # this rv will raise DummyException
+    @patch(fqname(Inlet), spec=Inlet, _pull=pull_mock(DummyIterable()))
     def test_generic_error_raised(self, inlet1):
         logging.getLogger('databay.Link').setLevel(logging.ERROR)
-        link = Link([inlet1], [], timedelta(seconds=1), name='test_generic_error_raised')
+        link = Link([inlet1], [], timedelta(seconds=1),
+                    name='test_generic_error_raised')
         # with self.assertRaisesRegex(TypeError, databay.link._ITERABLE_EXCEPTION):
         self.assertRaises(DummyException, link.transfer)
 
-    @patch(fqname(Inlet),spec=Inlet)
-    def test_integer_to_timedelta(self, inlet1):
-        link = Link([inlet1], [], 1, name='test_integer_interval_coerced')
+    def test_integer_to_timedelta(self):
+        link = Link([], [], 1, name='test_integer_interval_coerced')
         self.assertEquals(link._interval, timedelta(seconds=1))
-    
-    @patch(fqname(Inlet),spec=Inlet)
-    def test_float_to_timedelta(self, inlet1):
-        link = Link([inlet1], [], 1.5, name='test_float_interval_coerced')
+
+    def test_float_to_timedelta(self):
+        link = Link([], [], 1.5, name='test_float_interval_coerced')
         self.assertEquals(link._interval, timedelta(seconds=1.5))
 
     @patch(fqname(Outlet), spec=Outlet)
     @patch(fqname(Inlet), spec=Inlet, _pull=pull_mock())
-    @patch(fqname(Inlet), spec=Inlet, _pull=pull_mock(object())) # this rv is invalid, should be a list
+    # this rv is invalid, should be a list
+    @patch(fqname(Inlet), spec=Inlet, _pull=pull_mock(object()))
     def xtest_non_iterable_caught(self, inlet1, inlet2, outlet1):
         logging.getLogger('databay.Link').setLevel(logging.CRITICAL)
-        link = Link([inlet1, inlet2], [outlet1], timedelta(seconds=1), name='test_non_iterable_caught', catch_exceptions=True)
+        link = Link([inlet1, inlet2], [outlet1], timedelta(
+            seconds=1), name='test_non_iterable_caught', catch_exceptions=True)
         results = asyncio.run(inlet2._pull(None))
         link.transfer()
         outlet1._push.assert_called_with(results, mock.ANY)
 
     @patch(fqname(Outlet), spec=Outlet)
     @patch(fqname(Inlet), spec=Inlet, _pull=pull_mock())
-    @patch(fqname(Inlet), spec=Inlet, _pull=pull_mock(DummyIterable())) # this rv will raise DummyException
+    # this rv will raise DummyException
```

Comment:
I can see your formatter has gone over and broke down the longer lines down. While we don't have any formatting decided on yet I think it's fine, although I'd be careful for situations like the one highlighted. The comment was meant for the line above, which after the breaking looks like it describes the line below it (although this is me assuming single-line comments describe lines below them - that could be subjective). 

This has happened a few times in this file, so could I suggest to go over the comments and move them one line above so both your formatter is happy, and the comments stick true to what they're describing? Maybe I'd even rephrase them or add a colon at the end to make it nonambiguous which lines they refer to. Please feel free to share your thoughts about this if you'd like to suggest otherwise 😊

Reply:
Yeah this was actually a mistake, I thought I had my VSCode settings set to ignore everything in my databay workspace but it went ahead and did this anyway. I'm happy to actually leave them as they were before the formatting kicked in.